CometServer/appsettings.Development.json

```diff
@@ -60,8 +60,9 @@
       "IsEnabled": false,
       "ValidIssuer": "CDP4-COMET",
       "ValidAudience": "localhost:5000",
-      "SymmetricSecurityKey": "needs-to-be-updated-with-a-secret",
-      "TokenExpirationMinutes": 150
+      "SymmetricSecurityKey": "needs-to-be-updated-with-a-secret", 

```

Comment:
please make sure to update the wiki with this information

Reply:
Configuration page updated, gonna update ICD